utils/kind-with-registry.sh

```diff
@@ -67,7 +67,7 @@ nodes:
 containerdConfigPatches:
 - |-
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
-    endpoint = ["http://${reg_host}:5000"]
+    endpoint = ["http://${reg_host}:${reg_port}"]
```

Comment:
I don't think adjustments are needed here.
Have you tested it? They use addresses inside the container

Reply:
My misunderstanding. Had reverted the script.